pilot/pkg/networking/core/v1alpha3/listener_test.go

```diff
@@ -135,10 +197,10 @@ func isHTTPListener(listener *xdsapi.Listener) bool {
 	return false
 }
 
-func buildService(protocol model.Protocol, creationTime time.Time) *model.Service {
+func buildService(hostname string, protocol model.Protocol, creationTime time.Time) *model.Service {
 	return &model.Service{
 		CreationTime: creationTime,
-		Hostname:     "*.example.org",
+		Hostname:     model.Hostname(hostname),
 		Address:      "1.1.1.1",
```

Comment:
Needs to be customizable.. When checking TCPWithCurrentHTTP, or HTTPWithCurrentTCP, you should always supply 0.0.0.0 as the address because conflicts occur only for these.

But with TCPWithCurrentTCP, you can test with any combo and ensrue that there is conflict for same ip:port pair

Reply:
done.